charts/cloudquery/templates/admin-deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       {{- toYaml .Values.deploymentAnnotations | nindent 8 }}
       {{- end }}
     spec:
-      {{- if .Values.serviceAccount.enabled }}
+      {{- if ne (include "cloudquery.serviceAccountName" .) "default" }}
```

Comment:
OLD:
If it's `false` then no `serviceAccount` directive is included post-templating

NEW:

If it's `false` and (`.Values.serviceAccount.name` == `""`) then no `serviceAccount` directive is included post-templating.

Reply:
If it's `true` then a serviceAccount is generated and that is used. 
If it's `false` but a `.Values.serviceAccount.name` is defined it attempts to use the serviceAccount.name provided

Reply:
If it's false but the name is defined, shouldn't it skip the SA altogether still?
:think:

Reply:
No, because we want people to be able to use pre-defined SAs.
Without this change, we'd need either two booleans or a ternary expression.

This method is more common in Helmcharts.